.github/workflows/UpdateContentOverview.yml

```diff
@@ -21,17 +21,17 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'npm'
       - run: npm install
-      - run: npm run update:widgets-overview
+      - run: npm run update:content-overview
```

Comment:
Can you name every step in the pipeline? All the others GH actions contain it.

Reply:
up to you

Reply:
Addedd